doc/release/platforms/README.sgi.rst

```diff
@@ -49,8 +55,7 @@ This can be done using the following settings:
 
```

Comment:
*Offline, I agreed to improve the warning tomorrow, and commit what's here in the meantime